sdk/cosmosdb/cosmos/review/cosmos.api.md

```diff
@@ -1452,10 +1480,11 @@ export type OperationWithItem = OperationBase & {
 
 // @public (undocumented)
 export interface PartitionedQueryExecutionInfo {
+    hybridSearchQueryInfo?: HybridSearchQueryInfo;
     // (undocumented)
     partitionedQueryExecutionInfoVersion: number;
```

Comment:
I noticed a bunch of shipped and new undocumented code - is that expected?

Reply:
There is some undocumented code in the SDK that has been shipped and left unaddressed for quite some time. We will review these undocumented areas and work on resolving them. However, for newer additions, we have ensured proper documentation is in place.